app/models/course/assessment/submission.rb

```diff
@@ -79,10 +79,6 @@ class Course::Assessment::Submission < ApplicationRecord
   has_many :question_bundle_assignments, class_name: Course::Assessment::QuestionBundleAssignment.name,
                                          inverse_of: :submission, dependent: :destroy
 
-  # @!attribute [r] graders
-  #   The graders associated with this submission.
-  has_many :graders, through: :answers, class_name: User.name
```

Comment:
Ooo why was this removed?

Reply:
That’s not used at all in the codebase and getting graders using this is very slow..

Reply:
Roger, thanks for clarifying